druid/src/scroll_component.rs

```diff
@@ -17,15 +17,18 @@
 
 use std::time::Duration;
 
-use crate::kurbo::{Point, Rect, Vec2};
 use crate::theme;
 use crate::widget::{Axis, Viewport};
+use crate::{
+    kurbo::{Point, Rect, Vec2},
+    Key,
+};
 use crate::{Env, Event, EventCtx, LifeCycle, LifeCycleCtx, PaintCtx, RenderContext, TimerToken};
```

Comment:
```suggestion
use crate::kurbo::{Point, Rect, Vec2};
use crate::{Env, Event, EventCtx, Key, LifeCycle, LifeCycleCtx, PaintCtx, RenderContext, TimerToken};
```

and `cargo fmt` after this